uaclient/cli/enable.py

```diff
@@ -216,7 +186,9 @@ def action_enable(args, *, cfg, **kwargs) -> int:
             if json_output:
                 progress = api.ProgressWrapper()
             else:
-                progress = api.ProgressWrapper(CLIEnableProgress())
+                progress = api.ProgressWrapper(
```

Comment:
Even though this is not a problem introduced in this PR, but I noticed that we are now creating the same `entitlement` object on lines 155 and 193. I think we should only do that once

Reply:
Yep I can consolidate them